modules/wri_event/config/install/field.field.node.event.field_calendar_description.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_calendar_description
+    - filter.format.basic_html
+    - node.type.event
+  module:
+    - text
+id: node.event.field_calendar_description
+field_name: field_calendar_description
+entity_type: node
+bundle: event
+label: 'Calendar Description'
+description: ''
```

Comment:
Let's give this a more helpful description:
```
The event description in the "Add to Calendar" link. Default value is "[node:title] - from WRI: [node:url:absolute]"
```
(you'll have to html encode the brackets so they look right and can be copy/pasted)